validator/client/service.go

```diff
@@ -22,12 +23,14 @@ type ValidatorService struct {
 	conn      *grpc.ClientConn
 	endpoint  string
 	withCert  string
+	p2p       p2p.Broadcaster
```

Comment:
The validator client will touch p2p ?

Reply:
In this design, yes. It's still under debate, I suppose, but this would be a trivial change if we wanted to offload broadcasting to the beacon chain node.